ml-agents-envs/mlagents_envs/communicator.py

```diff
@@ -8,8 +8,8 @@ def __init__(self, worker_id=0, base_port=5005):
         """
         Python side of the communication. Must be used in pair with the right Unity Communicator equivalent.
 
+        :int worker_id: Offset from base_port. Used for training multiple environments simultaneously.
```

Comment:
Rearranged to match the argument order. I think I changed the wording of this wherever I saw it (`git blame` showed it was 3 years old).